cli/src/utils.rs

```diff
@@ -105,9 +99,9 @@ impl std::str::FromStr for MetadataVersion {
 impl From<MetadataVersion> for CodegenMetadataVersion {
     fn from(input: MetadataVersion) -> CodegenMetadataVersion {
         match input.version {
-            14 => CodegenMetadataVersion::V14,
             u32::MAX => CodegenMetadataVersion::Unstable,
-            _ => panic!("MetadataVersion and CodegenMetadataVersion are not in sync!"),
+            14 => CodegenMetadataVersion::Latest,
```

Comment:
Nit: maybe just manually have an `into_codegen_metadata_version(metadata_version: Option<MetadataVersion>) -> CodegenMetadataVersion` so we don't have to hardcode the 14 here?

Reply:
Or clap has `default_value = <str>` which we could use so that `MetadataVersion` could have a `Latest` variant and clap will default to using that. (maybe we can just use `CodegenMetadataVersion` directly in that case?)

Reply:
That makes sense! Have ended up using just the `CodegenMetadataVersion` here :D Thanks!